AUTHORING_GUIDE.md

```diff
@@ -299,198 +151,124 @@ If documenting primitive types, be sure to note if they have a particular set
-    * gae(sample='./appengine/standard/storage/appengine-client')
-    * lint(sample='./appengine/flexible/storage')
-    * lint(sample='./appengine/standard/storage/api-client')
-    * lint(sample='./appengine/standard/storage/appengine-client')
-    * lint(sample='./storage/api')
-    * lint(sample='./storage/cloud-client')
-    * lint(sample='./storage/transfer_service')
-    * py27(sample='./appengine/flexible/storage')
-    * py27(sample='./storage/api')
-    * py27(sample='./storage/cloud-client')
-    * py35(sample='./appengine/flexible/storage')
-    * py35(sample='./storage/api')
-    * py35(sample='./storage/cloud-client')
-    * readmegen(sample='./storage/api')
-    * readmegen(sample='./storage/cloud-client')
-    * readmegen(sample='./storage/transfer_service')
+```
+$ nox -k storage -l
+* gae(sample='./appengine/standard/storage/api-client')
+* gae(sample='./appengine/standard/storage/appengine-client')
+* lint(sample='./appengine/flexible/storage')
+* lint(sample='./appengine/standard/storage/api-client')
+* lint(sample='./appengine/standard/storage/appengine-client')
+* lint(sample='./storage/api')
+* lint(sample='./storage/cloud-client')
+* lint(sample='./storage/transfer_service')
+* py27(sample='./appengine/flexible/storage')
+* py27(sample='./storage/api')
+* py27(sample='./storage/cloud-client')
+* py35(sample='./appengine/flexible/storage')
+* py35(sample='./storage/api')
+* py35(sample='./storage/cloud-client')
+* readmegen(sample='./storage/api')
+* readmegen(sample='./storage/cloud-client')
+* readmegen(sample='./storage/transfer_service')
+```
 
 Now you can use nox to run a specific session, for example, if you want to lint
 the storage cloud-client samples:
 
-    $ nox -s "lint(sample='./storage/cloud-client')"
+```
+$ nox -s "lint(sample='./storage/cloud-client')"
+```
 
-### Test environment setup
+## Test Environment Setup
 
-Because all the tests here are system tests, you'll need to have a Google
-Cloud project with billing enabled. Once you have this configured, you'll
-need to set environment variables for the tests to be able to use your project
-and its resources. See `testing/test-env.tmpl.sh` for a list of all environment
-variables used by all tests. Not every test needs all of these variables.
+Because all tests are system tests that use live resources, running tests requires a Google Cloud project with billing enabled, as covered under [Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
 
-#### Google Cloud Storage resources
+Once you your project created and configured, you'll need to set environment variables to identify the project and resources to be used by tests. See [testing/test-env.tmpl.sh](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/testing/test-env.tmpl.sh) for a list of all environment variables used by all tests. Not every test needs all of these variables.
```

Comment:
s/Once you/Once you have/

Reply:
Missed this one, resolving in next commit.